modules/auxiliary/scanner/ssl/ssl_version.rb

```diff
@@ -472,7 +472,7 @@ def run_host(ip)
     begin
       ctx = { 'Msf' => framework, 'MsfExploit' => self }
       # Initialize rex-sslscan scanner
-      scanner = Rex::SSLScan::Scanner.new(ip, rport, ctx)
+      scanner = Rex::SSLScan::Scanner.new(ip, rport, ctx, tls_server_name_indication: datastore['RHOSTNAME'])
```

Comment:
You'll want to add this to the datastore options so it appears to the user when they run `show options` command as well adding validation etc, similar to this module:

https://github.com/rapid7/metasploit-framework/blob/9efd3c3c790fe1ba49349b1ed0991536576de618/lib/msf/core/exploit/remote/tcp.rb#L66